main/src/main/scala/org/clulab/processors/clu/tokenizer/SentenceSplitter.scala

```diff
@@ -186,6 +196,10 @@ class SpanishSentenceSplitter extends RuleBasedSentenceSplitter {
 object SentenceSplitter {
   val EOS: Regex = """^[\.!\?\s]+$""".r
 
+  // Control string that enforces a sentence break
+  // If you change this value, change also the SENTENCEBREAK in OpenDomainLexer.g to the same value (and recompile the Antlr grammar)
+  val SENTENCE_BREAK_CONTROL_STRING = "[SB]"
```

Comment:
So the procedure would be to figure out before tokenization, so before `Processor.mkDocument`, where the control strings should be, like where there's a `<br>`, and change them to `[SB]`.  These two strings happen to be the same length and one could take the resulting `Document` and substitute the old text for the new text in order to preserve the original.  If the strings are different lengths, all the offsets would be off and the substitution won't work.  We would lose (easy) access to the original document text.  Will that be a problem?

Reply:
I think it is likely that this will change offsets (e.g., when replacing newlines with '[SB]'). Users need to be aware of this.

Reply:
I'm happy with it just being off by default.  If someone turns it on, (a simple `SentenceSplitter.useControlString = true`) and it's important, they can be responsible for making sure that the control string is not already in their text and if necessary, escaping it before and unescaping it after, etc.

We do in general have cases in which provenance is important and the original text needs to be preserved.  This new feature is still useful and can be used when the original is not so important, though.